### PR TITLE
Add max-store-preparing-time in pd-ctl

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9502 https://github.com/pingcap/docs-cn/pull/9604 https://github.com/pingcap/docs-cn/pull/9637
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by rleungx — APPROVED** [review]

File: pd-control.md [hunk 1]
```diff
@@ -244,6 +244,14 @@ Usage:
     >> config set max-store-down-time 30m  // Set the time within which PD receives no heartbeats and after which PD starts to add replicas to 30 minutes
     ```
 
+- `max-store-preparing-time` controls the maximum waiting time for the store to go online. During the online stage of a store, PD can query the online progress of the store. When the specified time is exceeded, PD assumes that the store has been online and cannot query the online progress of the store again.
+
+    The following command specifies that the maximum waiting time for the store to go online is 4 hours.
+
+    ```bash
+    >> config set max-store-preparing-time 4h
+    ```
```
Comment (Oreoxmt) [inline pd-control.md:253]:
```suggestion
    {{< copyable "" >}}
    
    ```bash
    >> config set max-store-preparing-time 4h
    ```
```

File: pd-configuration-file.md [hunk 2]
```diff
@@ -213,6 +213,11 @@ Configuration items related to scheduling
 + The downtime after which PD judges that the disconnected store can not be recovered. When PD fails to receive the heartbeat from a store after the specified period of time, it adds replicas at other nodes.
 + Default value: `30m`
 
+### `max-store-preparing-time` <span class="version-mark">New in v6.1.0</span>
+
++ Controls the maximum waiting time for the store to go online. During the online stage of a store, PD can query the online progress of the store. When the specified time is exceeded, PD assumes that the store has been online and cannot query the online progress of the store again. But this does not prevent Regions from transferring to the newly online store. In most scenarios, you do not need to adjust this parameter.
```
Comment (Oreoxmt) [inline pd-configuration-file.md:218]:
```suggestion
+ Controls the maximum waiting time for the store to go online. During the online stage of a store, PD can query the online progress of the store. When the specified time is exceeded, PD assumes that the store has been online and cannot query the online progress of the store again. But this does not prevent Regions from transferring to the new online store. In most scenarios, you do not need to adjust this parameter.
```

File: pd-control.md [hunk 3]
```diff
@@ -244,6 +244,16 @@ Usage:
     >> config set max-store-down-time 30m  // Set the time within which PD receives no heartbeats and after which PD starts to add replicas to 30 minutes
     ```
 
+- `max-store-preparing-time` controls the maximum waiting time for the store to go online. During the online stage of a store, PD can query the online progress of the store. When the specified time is exceeded, PD assumes that the store has been online and cannot query the online progress of the store again. But this does not prevent Regions from transferring to the newly online store. In most scenarios, you do not need to adjust this parameter.
```
Comment (Oreoxmt) [inline pd-control.md:247]:
```suggestion
- `max-store-preparing-time` controls the maximum waiting time for the store to go online. During the online stage of a store, PD can query the online progress of the store. When the specified time is exceeded, PD assumes that the store has been online and cannot query the online progress of the store again. But this does not prevent Regions from transferring to the new online store. In most scenarios, you do not need to adjust this parameter.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: pd-control.md [hunk 4]
```diff
@@ -244,6 +244,15 @@ Usage:
     >> config set max-store-down-time 30m  // Set the time within which PD receives no heartbeats and after which PD starts to add replicas to 30 minutes
     ```
 
+- `max-store-preparing-time` controls the maximum waiting time for the store to go online. During the online stage of a store, PD can query the online progress of the store. When the specified time is exceeded, PD assumes that the store has been online and cannot query the online progress of the store again. But this does not prevent Regions from transferring to the new online store. In most scenarios, you do not need to adjust this parameter.
+
+    The following command specifies that the maximum waiting time for the store to go online is 4 hours.
+
+    {{< copyable "" >}}
+    ```bash
```
Comment (Oreoxmt) [inline pd-control.md:252]:
```suggestion

    ```bash
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**